src/Option.js

```diff
@@ -79,18 +79,27 @@ class Option extends React.Component {
 				{this.props.children}
 			</div>
 		) : (
-			<div className={className}
-				style={option.style}
-				role="option"
-				onMouseDown={this.handleMouseDown}
-				onMouseEnter={this.handleMouseEnter}
-				onMouseMove={this.handleMouseMove}
-				onTouchStart={this.handleTouchStart}
-				onTouchMove={this.handleTouchMove}
-				onTouchEnd={this.handleTouchEnd}
-				id={instancePrefix + '-option-' + optionIndex}
-				title={option.title}>
-				{this.props.children}
+			<div className={this.props.isUniqueSelected ? className : null}>
+				{this.props.isUniqueSelected &&
+					(<span className="Select-unique-selected-icon-wrapper">
```

Comment:
Could you put this `(` in the previous line, so that the markup is in its own line?